src/core/binding.c

```diff
@@ -1652,7 +1632,7 @@ QuicBindingReceive(
         DatagramChain = Datagram->Next;
         Datagram->Next = NULL;
 
-        QUIC_RX_PACKET* Packet = GetQuicRxPacket(Datagram);
+        QUIC_RX_PACKET* Packet = (QUIC_RX_PACKET*)Datagram;
```

Comment:
It looks like we can completely get rid of the notion of datagram in this function now.

Reply:
We could, but I think it would add a lot of casts back and forth for all the chains that are built. The is the top level function from the platform layer, so it has to receive datagrams. For now, I was just going to leave this as is.